integration-test/src/main/java/com/sequenceiq/it/util/cleanup/CleanupUtil.java

```diff
@@ -172,21 +191,213 @@ public void cleanupCredentials() {
         return sdxNames;
     }
 
-    private void deleteEnvironments(Set<String> environmentNames, EnvironmentClient environmentClient) {
-        environmentNames.forEach(environmentName -> LOG.info("Environment with name: {} will be deleted!", environmentName));
+    private List<String> getResourcesFromFile(String resourceNameType, Path filePath) {
+        List<String> resourceNames = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath.toString()))) {
+            StringBuilder builder = new StringBuilder();
+            String line;
+            while ((line = reader.readLine()) != null) {
```

Comment:
Please use this standard library method for file reading:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#readString(java.nio.file.Path)